docs/developer.md

```diff
@@ -38,7 +38,7 @@ which removes everything.
 
 ### Configuration
 
-Create a `config/config.yaml` and `config/dex.yml` config file based on their `config/*.dist` counterparts with:
+Create a `config/config.yaml` and `etc/config/dex.yml` config file based on their `config/*.dist` counterparts with:
 
 ```bash
 $ make config/config.yaml config/dex.yml
```

Comment:
I assume the make commands are also outdated. Can you please check them?

Reply:
They are ok, it was wrong only in the documentation.

Reply:
```suggestion
$ make config/config.yaml etc/config/dex.yml
```